loom/engine/loom2d/l2dRectangle.h

```diff
@@ -260,15 +260,18 @@ class Rectangle
     /**
      * Make a copy of this Rectangle.
      */
-    Rectangle *clone()
+    int clone(lua_State *L)
```

Comment:
Why is this the right change? ie what does this accomplish that the normal memory mgmt didn't?